awx/main/management/commands/replay_job_events_processing.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+import socket
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+from awx.main.tasks.callback import RunnerCallback
+from awx.main.models import Job, JobHostSummary
+from awx.main.constants import ACTIVE_STATES
+
+import ansible_runner
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('job_id', type=int)
+
+    def handle(self, *args, **options):
+        runner_callback = RunnerCallback(model=Job)
+        job = Job.objects.get(pk=options['job_id'])
+        runner_callback.instance = job
+        runner_callback.job_created = str(job.created)
+
+        job.job_events.all().delete()
```

Comment:
@AlanCoding @fosterseth What's the fastest way to delete job events for a particular job?